src/components/InstallTabs/InstallTabs.scss

```diff
@@ -92,21 +92,18 @@
         user-select: none;
       }
     }
-    &__docs-button {
+    &__docs-link {
+      box-sizing: border-box;
+      display: block;
       border-radius: var(--space-04);
```

Comment:
Quick note: if you decide you don't want to style this link similar to a browser default button, then I can remove `border-radius` as it is no longer needed. 👍